compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEHoistForAffineApply.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree-amd-aie/Transforms/Passes.h"
+#include "iree-amd-aie/Transforms/Transforms.h"
+#include "mlir/Dialect/Affine/IR/AffineOps.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
+
+namespace mlir::iree_compiler::AMDAIE {
+
+LogicalResult hoistForAffineApplyOp(RewriterBase &rewriter,
```

Comment:
Whenever we implement a pass it's a good practice to add `DEBUG_TYPE` - similar to [this](https://github.com/nod-ai/iree-amd-aie/blob/71834c279050348faf747472747efa04afea0df2/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp#L21).

Reply:
Thanks, done.